### ISSUE TITLE
migrations and archives... 

### ISSUE THREAD

**Issue by marcoscaceres** [WICG/admin#175]
- [x] * Element Capture migrated from https://github.com/WICG/element-capture to https://github.com/screen-share/element-capture
- [x] * https://github.com/WICG/multicapture/issues/7 (moved)
- [x] * https://github.com/screen-share/capture-all-screens/issues/8 (migrate to another CG)
- [x] * transfer-size can be archived according to https://lists.w3.org/Archives/Public/public-wicg/2023Feb/0009.html
- [x] * floc can be archived according to https://lists.w3.org/Archives/Public/public-wicg/2023Feb/0009.html
- [ ] * https://github.com/WICG/display-override can be archived according to https://lists.w3.org/Archives/Public/public-wicg/2023Feb/0016.html
- [ ] *  https://github.com/WICG/file-handling can be archived as it's  https://wicg.github.io/manifest-incubations/#file_handlers-member
- [ ] * https://github.com/WICG/lock-screen can be archived according to https://lists.w3.org/Archives/Public/public-wicg/2023Feb/0016.html
- [ ] * https://github.com/WICG/pwa-url-handler can be probably be archived as it's moved to  https://github.com/WICG/web-app-launch (needs confirmation)
- [ ] *   https://github.com/WICG/raw-clipboard-access can be archive according to https://lists.w3.org/Archives/Public/public-wicg/2023Feb/0016.html

**Comment by yoavweiss** [comment]
Regarding ones that can be archived, should we try and have the repo owners close or migrate issues before archiving?

**Comment by marcoscaceres** [comment]
Yes, it would be prudent to ask. There are some that seem ok to just archive though. Will be sure to check on a case by case basis.
